src/method.jl

```diff
@@ -118,7 +118,7 @@ function percival(
   ω = 1.0
 
   # create initial subproblem
-  al_nlp = AugLagModel(nlp, y, T(μ), x, cons(nlp, x) - nlp.meta.lcon)
+  al_nlp = AugLagModel(nlp, y, T(μ), x, obj(nlp, x), cons(nlp, x) - nlp.meta.lcon)
```

Comment:
`fx = obj(nlp, x)` was computed above.
A couple of lines below `fx` is computed manually, but it looks like we can use `nlp.fx` now.